README.md

```diff
@@ -1 +1,8 @@
 # gluster-mixins
+## Prerequisites
+1. Jsonnet
+
+## Install
+1. Clone this repo
```

Comment:
can you give a link.
`git clone https://github.com/gluster/gluster-mixins`

Reply:
ack.